pkgs/development/tools/vend/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule {
+  pname = "vend";
+  version = "unstable-2020-06-04";
+
+  # A permanent fork from master is maintained to avoid non deterministic go tidy
```

Comment:
can we just fetch https://github.com/c00w/vend/commit/49899829b1e4bcd22ee3ba2d12bc76863196cb03 as a patch?

Reply:
I don't mind if we had our own fork so we can make sure it does not break anything in nixpkgs.

Reply:
*shrug* flipped to patch.